setup.py

```diff
@@ -10,7 +10,7 @@
     'SKIP_PYTHON_MODULES' not in os.environ and
     'SKIP_PYTHON_SCRIPTS' not in os.environ
 ):
-    install_requires.append("distro; python_version >= '3.8'")
+    install_requires.append("distro >= 1.4.0; python_version >= '3.8'")
```

Comment:
@cottsay I've never combined a package version constraint and a python version constraint. This parsed and worked for me locally on Arch Linux but I'd like a second look in case I'm missing a subtlety or style thing.

Reply:
A first for me, too. Seems right, though :+1: